### ISSUE TITLE
session still exist after logout

### ISSUE THREAD

**Issue by redcoder77** [sapienza/CodeIgniter-admin-panel#20]
i downloaded the code and tried it out.
After logged in and logout , the previous session  still exist, which mean if I press the browser Back button after logout, I still can view the previous page.

**Comment by saikatharryc** [comment]
no,, try to logout and then close your browser  and then open, sometimes browsers not reloads the page just shows you from cache memory or from history.try by reloading the page after logout. if not you have still confusion , try to make an operation after loogout on previous page. it should require to log in.
